lineapy/transformer/node_transformer.py

```diff
@@ -661,6 +665,32 @@ def visit_Attribute(self, node: ast.Attribute) -> CallNode:
             self.visit(ast.Constant(value=node.attr)),
         )
 
+    if sys.version_info >= (3, 8):
+
```

Comment:
This should not be required. Since NamedExp was introduced in 3.8+, the visitor can just sit there without ever being called in 3.7 runtime env.

Reply:
you might need to escape this for typing, in which case you can do something similar to visit_Ellipsis

Reply:
When I try to use something similar to visit_Ellipsis, and try to run `pytest tests/end_to_end/test_op.py -v`, then I receive the following error:
```
ImportError while loading conftest '/Users/aayankumar/Desktop/linea/lineapy/conftest.py'.
conftest.py:3: in <module>
    from lineapy.cli.cli import setup_ipython_dir
lineapy/__init__.py:6: in <module>
    from lineapy.editors.ipython import start, stop, visualize
lineapy/editors/ipython.py:21: in <module>
    from lineapy.transformer.node_transformer import transform
lineapy/transformer/node_transformer.py:106: in <module>
    class NodeTransformer(ast.NodeTransformer):
lineapy/transformer/node_transformer.py:668: in NodeTransformer
    def visit_NamedExpr(self, node: ast.NamedExpr) -> Any:
E   AttributeError: module 'ast' has no attribute 'NamedExpr'
```

Looks like while importing `node_transformer.py`, Python tries to import this method as well, and since Python 3.7 does not have `ast.NamedExpr`, it causes an error. So for type purposes, will have to use this approach.

Reply:
hmm i see ast.NamedExpr itself is new.. okay lets go with what you have in that case!